### ISSUE TITLE
CI

### ISSUE THREAD

**Issue by mnivoliez** [relamptk/vr-streaming-overlay#13]
Hello, it's me again. I was wondering if you got any kind of continuous integration or if you plan to set it up (for exemple to build automatically new release or even master)?

**Comment by beniwtv** [comment]
Yes, eventually I want to get that set up, just haven't had the time yet.

**Comment by beniwtv** [comment]
I recently set up GitHub CI for godot-openvr, so this should not be that difficult now :)
